### PR TITLE
TCD/JDJ/RULE19-21

### PR DESCRIPTION
**Pull request by JacksonJ-KC**

### CODE REVIEW

File: rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_21.json [hunk 1]
```diff
@@ -0,0 +1,3109 @@
...
+                            }
+                        ]
+                    }
+                ],
+                "weather": {
+                    "climate_zone": "CZ5A"
+                }
+            },
+            "baseline": {
+                "id": "ASHRAE229 1",
```
Comment (yunjoonjung-PNNL) [inline rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_21.json:167]: Could we have ```weather``` and ```climate_zone``` keys (here and also in the following TCDs?

Comment (JacksonJ-KC) [reply rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_21.json:167]: the weather and climate_zone are on lines 307 & 308. They are showing up as the last objects within the baseline/proposed RMDs

**Comment by yunjoonjung-PNNL** [comment]
Could you check systems 7 and 10 throughout the TCDs using the method I let you know via email? This is because these systems aren't recognized in the ```get_baseline_system_types``` function.

File: rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_21.json [hunk 2]
```diff
@@ -0,0 +1,3105 @@
...
+                                "child_loops": [
+                                    {
+                                        "id": "Secondary CHW Loop 1",
+                                        "type": "COOLING"
+                                    }
+                                ]
+                            }
+                        ]
+                    }
+                ],
```
Comment (yunjoonjung-PNNL) [inline rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_21.json:162]: The ```schedules``` key's missing. Could you add proper schedules here and also in other cases?

Comment (yunjoonjung-PNNL) [reply rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_21.json:162]: I believe we still need to have the ```schedules``` to find the max heating thermostat setpoint values. What's not clear to me the schema doesn't specify what constant value it has to be. I believe the description should be updated.

Comment (JacksonJ-KC) [reply rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_21.json:162]: I interpreted it to mean if `thermostat_heating_setpoint_schedule` or `thermostat_cooling_setpoint_schedule` are not provided, then `design_thermostat_heating_setpoint` or `design_thermostat_cooling_setpoint` are assumed to be constant

Comment (yunjoonjung-PNNL) [reply rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_21.json:162]: Yeah good point! The thing is RDS doesn't specify what the constant value should be when the schedule doesn't exist. Let me ask this question to the RDS team (I'll copy you).

**Comment by yunjoonjung-PNNL** [comment]
@jugonzal07 I've one question. Can ```hourly_values``` under the ```schedules``` be pint quantities? It seems like heating temperature setpoint values aren't converted to pint quantities. I'd appreciate it if you could let us know!

@JacksonJ-KC TCDs are in the SI units. 70 degC seems very high for heating setpoint. I believe you input in the IP unit. Could you specify the units in the Excel file or input the values in SI unit only for the temperature schedule field?

**Comment by yunjoonjung-PNNL** [comment]
Closed as the latest TCD is available in PR #1076.

**Pull request closed without merging**